src/product/product-action.js

```diff
@@ -0,0 +1,83 @@
+import axios from "axios";
```

Comment:
If you are using axios you can create an instance of it in a separate file with configurations and then export that instance and use that in your action files, like:
```
// src/axios.js
import axios from "axios";

const instance = axios.create({
  baseURL: "http://localhost:4000"
});
export default instance;
```
so you don't need to put the baseUrl again and it's easy to change later as well